### PR TITLE
feat: Validate multiple scopes

### PR DESCRIPTION
**Pull request by pablosichert**
Hey there! Thanks for this cool app.

This PR allows validating multiple scopes, as in accordance with https://github.com/conventional-changelog/commitlint/pull/901.

### CODE REVIEW

**Comment by pablosichert** [comment]
Hey @zeke, pinging you to review, as suggested in your status 😄

File: lib/is-semantic-message.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = function isSemanticMessage (message, validScopes, validTypes, a
   }
 
   const [result] = commits
-  const { scope, type } = result.header
-  const isScopeValid = !validScopes || !scope || validScopes.includes(scope)
+  const { scope: scopes, type } = result.header
+  const isScopeValid = !validScopes || !scopes || scopes.split(',').every(scope => validScopes.includes(scope))
```
Comment (zeke) [inline lib/is-semantic-message.js:20]:
```suggestion
  const isScopeValid = !validScopes || !scopes || scopes.split(',').map(trim).every(scope => validScopes.includes(scope))
```

Comment (zeke) [reply lib/is-semantic-message.js:20]: ☝🏼 In case people include spaces after their commas.

**Review by zeke — COMMENTED** [review]
Looks good! I left a tiny suggestion, but I'm happy to ship this soon.

File: lib/is-semantic-message.js (hunk 1 above)
Comment (pablosichert) [reply lib/is-semantic-message.js]: Done and added test.

**Comment by zeke** [comment]
Nice! Thanks for your contribution, it should be deployed in the next few minutes. ✨

**Pull request merged**